jobs/v4/howto/auto_complete_sample_test.go

```diff
@@ -29,7 +29,7 @@ func TestJobTitleAutocomplete(t *testing.T) {
 	if _, err := jobTitleAutocomplete(buf, tc.ProjectID, companyID, "Software"); err != nil {
 		t.Fatalf("jobTitleAutoComplete: %v", err)
 	}
-	want := "Software Developer"
+	want := "Software Engineer"
```

Comment:
Was this working before and then broke because 'Developer' is no longer being returned in the top results?

To make this less brittle, I recommend updating the PageSize to 10. Seems reasonable that 'Software Developer' would show up in the top 10 and, if not, it might be a sign that the API is broken.

Else simply verify that the output includes a name with 'Software' in its title

Reply:
@anguillanneuf Yes the autocompletion ranking could change over time based on the job counts in live dataset, so I think updating the PageSize to 10 or verifying the output includes 'Software' would be better.

Sorry for a few back and forth in the past few weeks. We've already finalized our implementation so we should not expect more code sample test failures going forwards, otherwise it'll be considered as a bug on our end.

Reply:
I updated the PageSize to 10. I hope this fix is enough.